project/Build.scala

```diff
@@ -8,7 +8,7 @@ import com.typesafe.sbt.SbtGit._
 
 object Build extends Build {
 
-  val baseVersion = "5.1.0"
+  val baseVersion = "5.1.1"
```

Comment:
This should be done by the person cutting the release as well. Can you set to `5.1.1-SNAPSHOT` instead?